tests/cpu/transforms/builtins/BuiltinTransformRegistry_tests.cpp

```diff
@@ -403,4 +403,9 @@ OCIO_ADD_TEST(Builtins, version_2_3_validation)
     TestStyle("ACES-OUTPUT - ACES2065-1_to_CIE-XYZ-D65 - HDR-1000nit-REC2020-D60-in-REC2020-D65_2.0");
     TestStyle("ACES-OUTPUT - ACES2065-1_to_CIE-XYZ-D65 - HDR-2000nit-REC2020-D60-in-REC2020-D65_2.0");
     TestStyle("ACES-OUTPUT - ACES2065-1_to_CIE-XYZ-D65 - HDR-4000nit-REC2020-D60-in-REC2020-D65_2.0");
+
+    // ProPhotoRGB / ROMM RGB builtin transforms.
+    TestStyle("LINEAR-RIMM_to_ACES2065-1_BFD");
+    TestStyle("ROMM_to_CIE-XYZ-D65_BFD");
+    TestStyle("PROPHOTO-RGB-SRGB-GAMMA_to_ACES2065-1");
```

Comment:
Please revert this. The test I wanted you to add was in tests/cpu/transforms/BuiltinTransform_tests.cpp.

You could add the following around line 724:
```
    { "ROMM_to_CIE-XYZ-D65_BFD",
        { 1.0e-6f,
        { 0.5f,           0.4f,           0.3f,      0.248054f,   0.216383f,   0.124372f },
        { 0.03f,          0.02f,          0.01f,     0.00160897f, 0.00140735f, 0.000677473f } } }, // below the break pnt

    { "LINEAR-RIMM_to_ACES2065-1_BFD",
        { 1.0e-6f,
        { 0.5f,           0.4f,           0.3f,      0.47351069f, 0.39131449f, 0.29965645f } } }
```